charts/tempo-distributed/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: tempo-distributed
 description: Grafana Tempo in MicroService mode
 type: application
-version: 0.6.0
+version: 0.7.0
```

Comment:
i was intending to keep the major and minor version of the chart in sync with the tempo version.  so this would be 0.6.1.  is that a bad idea?

Reply:
As far as I know, minor updates are used for the patches / smaller changes. Major versions normally indicate there is a breaking change in the chart when you upgrade. The memcached update is a breaking change in this case.

Reply:
yeah, good point. keeping these in sync while playing nice with semver will probably be impossible in the long run.  may as well rip off the band aid now.  0.7.0 is good :+1: